libs/build_esma_cmake.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -eux
+
+name="esma_cmake"
+repo=${1:-${STACK_esma_cmake_repo:-"GEOS-ESM"}}
+version=${2:-${STACK_esma_cmake_version:-"main"}}
+id=${version//\//-}
+
+if $MODULES; then
+  set +x
+  source $MODULESHOME/init/bash
+  module list
+  set -x
+
+  prefix="${PREFIX:-"/opt/modules"}/core/$name/$repo-$id"
+  if [[ -d $prefix ]]; then
+    [[ $OVERWRITE =~ [yYtT] ]] && ( echo "WARNING: $prefix EXISTS: OVERWRITING!"; $SUDO rm -rf $prefix; $SUDO mkdir $prefix ) \
+                               || ( echo "WARNING: $prefix EXISTS, SKIPPING"; exit 1 )
+  fi
+else
+  prefix=${CMAKEMODULES_ROOT:-"/usr/local"}
```

Comment:
This is incorrect and should be `ESMA_CMAKE_ROOT`.

Reply:
Addressed in 2c049520a84a6938bc1724058b4e2d81c30376ae. Apologies for the multiple typos.